packages/server/src/fhircast/routes.ts

```diff
@@ -92,13 +94,36 @@ protectedRoutes.post(
       return;
     }
 
-    await getRedis().publish(req.params.topic as string, JSON.stringify(req.body));
+    const { event } = req.body as FhircastMessagePayload;
+    let stringifiedBody = JSON.stringify(req.body);
+    // Check if this an open event
+    if (event['hub.event'].endsWith('-open')) {
+      // TODO: Support this as a param for event type: "versionable"?
+      if (event['hub.event'] === 'diagnosticreport-open') {
+        event['context.versionId'] = generateId();
```

Comment:
https://confluence.hl7.org/display/FHIR/2020-09+FHIRcast+DiagnosticReport-update+Request+1

Should be `DiagnosticReport.meta.versionId`

Reply:
It seems like they might have done away with this requirement:

https://build.fhir.org/ig/HL7/fhircast-docs/3-6-3-DiagnosticReport-update.html#:~:text=The%20context.versionId%20matches%20the%20context.versionId%20provided%20by%20the%20Hub%20in%20the%20most%20recent%20DiagnosticReport%2Dopen%20or%20DiagnosticReport%2Dupdate%20event.

Reply:
Also see: 
https://build.fhir.org/ig/HL7/fhircast-docs/3-6-1-DiagnosticReport-open.html#:~:text=If%20a%20Hub%20supports%20content%20sharing%2C%20when,the%20context.versionId%20can%20be%20safely%20ignored.

Reply:
K, that works for me.  In the 2nd example, the `versionId` strings are different, so I guess that tells the story.